### ISSUE TITLE
yolo v3 custom object detection

### ISSUE THREAD

**Issue by Areej25890** [NSTiwari/YOLOv3-Custom-Object-Detection#1]
I have done every thing in the pdf but after training only classes.txt and testing.cfg files were generated. I mean the problem is that weights file after training wasn't generated. How can I solve this problem.

**Comment by NSTiwari** [comment]
@Areej25890, check if the `train.txt` file is empty or not. If it's empty, then the training probably didn't start; no `.weights` file will be generated thereby.

**Comment by Areej25890** [comment]
Yes. I found it empty and I have fixed it but I have another issue when I try to test it using object detection code I got the following error:
Index 1135 is out of bounds for axis 0 with size 100. What is the problem?. Thank you
________________________________

**Comment by Gilna-byte** [comment]
As I'm very new to this algorithm. I'd the same issue where @Areej25890 had. The weights file after training wasn't generated. How to check the train.txt is empty or not?

**Comment by Maharshi-joshi** [comment]
Hey @NSTiwari , could you help me solve this query:

yolo] params: iou loss: mse (2), iou_norm: 0.75, obj_norm: 1.00, cls_norm: 1.00, delta_norm: 1.00, scale_x_y: 1.00
Total BFLOPS 65.312 
avg_outputs = 516922 
 Allocate additional workspace_size = 12.46 MB 
Loading weights from darknet53.conv.74...
 seen 64, trained: 0 K-images (0 Kilo-batches_64) 
Done! Loaded 75 layers from weights-file 
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
 Detection layer: 82 - type = 28 
 Detection layer: 94 - type = 28 
 Detection layer: 106 - type = 28 
 If error occurs - run training with flag: -dont_show 
Unable to init server: Could not connect: Connection refused

(chart_yolov3_training.png:1513): Gtk-WARNING **: 07:03:54.744: cannot open display: 

This happens when I try to train the model at the last step.
